src/App.jsx

```diff
@@ -2,18 +2,20 @@ import "./App.css";
 import ListItem from "./components/ListItem";
 
 function App() {
+  const animals = [
```

Comment:
As this `animals` is a constant, don't forget the naming rule for a constant, try to use uppercase here.

Reply:
I agree that some constant variables that are global have to be named using uppercase however this particular variable is not global which is why it was not written as such.